CHANGELOG.md

```diff
@@ -3,6 +3,108 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.3.0
+
+_Jan 21, 2022_
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow to group rows based on column value (#3277) @flaviendelangle
+
+  ⚠️ This feature is temporarily available on the Pro plan until the release of the Premium plan.
+
+  To avoid future regression for users of the Pro plan, the feature needs to be explicitly activated using the rowGrouping experimental feature flag.
+
+  ```tsx
+  // To fully control
+  <DataGridPro
+    rowGroupingModel={rowGroupingModel}
+    onRowGroupingModel={newModel => setRowGroupingModel(newModel)}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+
+  // To initialize without controlling
+  <DataGridPro
+    initialState={{
+      rowGrouping: {
+        model: rowGroupingModel,
+      },
+    }}
+    experimentalFeatures={{ rowGrouping: true }}
+  />
+  ```
+
+  See the [documentation](https://mui.com/components/data-grid/group-pivot/#row-grouping) for more details.
+
+- ⚡ Add `is any of` filter operator (#2874) @alexfauquette
+- ✨ Introduce a `maxWidth` property in `GridColDef` (#3550) @flaviendelangle
```

Comment:
@flaviendelangle can you add an example for this one, please?

Reply:
- ✨ Introduce a `maxWidth` property in `GridColDef` (#3550) @flaviendelangle

  You can now limit the width of the flex columns and the resizable columns with the new `maxWidth` property on `GridColDef`.

  ```ts
  const columns: GridColDef[] = [
    { field: 'director', flex: 1, maxWidth: 200 }, // will take the free space up to 200px and will not be resizable above 200px
    { field: 'year', maxWidth: 150 }, // will not be resizable above 200px 
  ]
  ```